clients/drcachesim/simulator/caching_device.cpp

```diff
@@ -185,7 +185,8 @@ caching_device_t::request(const memref_t &memref_in)
             missed = true;
             // If no parent we assume we get the data from main memory
             if (parent_ != NULL) {
-                parent_->stats_->child_access(memref, false, cache_block);
+                for (caching_device_t *up = parent_; up != nullptr; up = up->parent_)
```

Comment:
(Optional) Worth extracting this loop into a separate routine? `update_child_access`?

Reply:
Done as part of changing the logic to propagate hits all the way but misses just one level.

Reply:
Unfortunately there's no way to test the miss logic b/c dr$sim does not care about child misses: it's something we want for external uses that subclasses this.  Prob sufficient to have just external tests.